lib/appsignal/error_handler.ex

```diff
@@ -32,12 +32,12 @@ defmodule Appsignal.ErrorHandler do
   def handle_event(event, state) do
     state =
      case match_event(event) do
-        {origin, reason, message, stack, metadata} ->
+        {origin, reason, message, stack, metadata, conn} ->
```

Comment:
You're right. I removed the metadata from `match_event/1` in 9579739ae382eb586f184da7f5a81395292d3fb2. As you can see, now it always calls `submit_transaction/6` with an empty map. 

If we'd change the attribute order in `submit_transaction/6` to `transaction, reason, message, stack, conn // nil, metadata // %{}`, we would drop the empty map in the call from `handle_event/2`. However, then it would mismatch the attribute order in `Appsignal.send_error/5`, where it's the other way around. 

We could change that too, but I'm not sure we want to change the API there to prevent having to use an empty map in `handle_event/2`. What do you think?